aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -387,9 +404,12 @@
     "/properties/MasterUsername",
     "/properties/NcharCharacterSetName",
     "/properties/Port",
+    "/properties/RestoreTime",
+    "/properties/SourceDBInstanceAutomatedBackupsArn",
```

Comment:
We had an issue https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/775 before that adding that parameter here will block customer from deleting this snapshot while it is not longer needed and customer have tight quota of snapshot. Should we move this to conditionalCreateOnly ?
@khebul can verify this as he resolved this here https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/pull/351